### PR TITLE
[PT-6477] Update and fix local rubocop

### PR DESCRIPTION
**Pull request by dlozano**
[PT-6477](https://kantox.atlassian.net/browse/PT-6477)

[PT-6477]: https://kantox.atlassian.net/browse/PT-6477?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by bestwebua — COMMENTED** [review]
Great, independent rubocop version! This is actually that I wanted to suggest. Maybe we can group and use not only rubocop linter, like action-linters with bunch of ruby related linters?

Just an example from my opensource project: https://github.com/truemail-rb/truemail/blob/master/.circleci/linter_configs/.lefthook.yml

File: Gemfile [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 gem 'rubocop-i18n', '~> 3.0.0'
-gem 'rubocop-rake', '~> 0.5.1' 
-gem 'pronto', '~> 0.10.0' 
-gem 'pronto-rubocop', github: 'prontolabs/pronto-rubocop', ref: '2c16f3f' #, '~> 0.10.0'
+gem 'rubocop-rake', '~> 0.6.0'
+gem 'rubocop-capybara', '~> 2.21.0'
+gem 'pronto', '~> 0.11.2'
+gem 'pronto-rubocop', '~> 0.11.5'
 
+gem 'faraday-retry', '~> 2.2.1'
```
Comment (bestwebua) [inline Gemfile:14]: For what we should include faraday here?

Comment (dlozano) [reply Gemfile:14]: `octokit` dependency. Needed to avoid warning.

Comment (bestwebua) [reply Gemfile:14]: @dlozano Got it, thanks for explanation!

**Comment by dlozano** [comment]
> Great, independent rubocop version! This is actually that I wanted to suggest. Maybe we can group and use not only rubocop linter, like action-linters with bunch of ruby related linters?
> 
> Just an example from my opensource project: https://github.com/truemail-rb/truemail/blob/master/.circleci/linter_configs/.lefthook.yml

Yes, we are also using brakeman, check our main project github actions. We can add additional stuff if needed.

**Review by bestwebua — APPROVED** [review]

**Pull request merged**